catalog.html

```diff
@@ -0,0 +1,266 @@
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="vissually-hidden">Избранное</span>
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a class="button" href="#">Хочу сюда!</a>
+          </li>
+        </ul>
+     </nav>
+
+    </header>
+
+    <main class="main-inner">
+
+      <section class="filter">
+
+      <header class="filter-header">
+        <h1 class="filter-header-title">Гостиницы Седоны</h1>
+        <ul class="breadcrumbs">
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link" href="index.html">
+              <img src="" alt="На главную">
+            </a>
+          </li>
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link">Гостиницы</a>
+          </li>
+        </ul>
+      </header>
+
+      <form class="filter-hotels" action="https://echo.htmlacademy.ru/" method="get">
+
+        <fieldset class="filter-hotels-group">
+          <legend class="filter-hotels-title">Инфраструктура:</legend>
+          <ul class="filter-hotels-list">
+            <li class="filter-hotels-item">
+              <label class="control">
+                <input class="control-input" type="checkbox" name="swimmingpool" checked>
+
+            Бассейн
+              </label>
+            </li>
+
+              <li class="filter-hotels-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="parking" checked>
+
+              Парковка
+                </label>
+              </li>
+
+              <li class="filter-hotels-item">
+                <label class="control">
+                  <input class="control-input" type="checkbox" name="wifi">
+
+              Wi-Fi
+                </label>
+              </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="filter-hotels-group">
+        <legend class="filter-hotels-title">Тип жилья:</legend>
+
+        <ul class="filter-hotels-list">
+          <li class="filter-hotels-item">
+            <label class="control">
+              <input class="control-input" type="radio" name="accomodation-type" value="hotel" checked>
+
+          Гостиница
+            </label>
+          </li>
+
+            <li class="filter-hotels-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="accomodation-type" value="motel">
+
+            Мотель
+              </label>
+            </li>
+
+            <li class="filter-hotels-item">
+              <label class="control">
+                <input class="control-input" type="radio" name="accomodation-type" value="apartment">
+
+            Апартаменты
+              </label>
+            </li>
+        </ul>
+      </fieldset>
+
+
+        <button class="button" type="submit">Применить</button>
+        <button class="button" type="submit">Сбросить</button>
```

Comment:
```suggestion
        <button class="button" type="reset">Сбросить</button>
```
Так как для сброса значений из полей формы используется кнопка с типом `reset`.